### DISCUSSION TITLE
Good for timeseries?

### DISCUSSION THREAD

**Discussion by wajihullahbaig** [Arize-ai/phoenix#676] [Q&A]
thank you for sharing such a wonderful too,
Right now I am working on time series data, trying to find clusters of user gathered up in a specified slice of timer performing an activity, will this tool help me in analysing time series?

**Comment by mikeldking** [comment]
Hey @wajihullahbaig 👋 Excited to have you here!

I'm guessing you have a timeseries forecasting model? Very cool!

So in the current incarnation of phoenix, we are definitely more positioned to work with unstructured data (things like text, image) so we might not be the ideal tool for you. We care building out tabular support as we speak though so would love to hear more about your use-case! We are currently building out drift and data quality visualizations on particular dimension slices.

I'm going to tag @statisticianinstilettos - she knows a ton about timeseries forecasting and might be able to support your use-case as well.

**Reply by wajihullahbaig** [reply]
Thanks for the detailed reply. I am using timeseries for a basic analysis - lets say for the first 10 seconds of the data, I want to know if the first set of K time series are likely behaving the same, like are they correlated in some sense

**Reply by axiomofjoy** [reply]
Hey @wajihullahbaig 👋 Thanks for checking out Phoenix. Can you tell us more about what signal you're measuring, perhaps with a concrete example? I'm also available if it's easier to talk it over in a quick video call, feel free to book a time on my [calendar](https://calendly.com/xander-arize-dev-advocate/30min).

**Reply by wajihullahbaig** [reply]
Hey @axiomofjoy - Ok let me make the problem as simple as possible as an example, what I have at hand are the heart rates from sensors. These heart rates a for female and male participants doing physical workouts. The point is, with this information, lets say I want to isolate participants having a high heart rate (lets say intense physical exercise ) vs low heart rate participants.
will it make sense or is it possible to use time series. Lets say we sample the heart rates for each second for 1 minute.

**Reply by axiomofjoy** [reply]
Got it. Do you mind sharing the format of your data (e.g., with a screenshot of `df.head()`) and maybe the modeling technique you're using? I'm not super deep on time-series modeling, this would help give me some context.

**Reply by wajihullahbaig** [reply]
Belated reply. I wish I could share the data with you guys. Really sorry. 
I have paused the work for now.

**Reply by axiomofjoy** [reply]
No worries, let us know if you pick it back up.
